microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -85,7 +86,7 @@ def verify_ltp_lite(
                 )
 
         # run ltp lite tests
-        ltp: Ltp = node.tools.get(Ltp, git_tag=ltp_tests_git_tag)
+        ltp: Ltp = node.tools.get(Ltp, ltp_prebuilt_file, git_tag=ltp_tests_git_tag)
```

Comment:
1. Use named parameters as much as possible. 
2. Make parameters names consistent as much as possible. It calls `ltp_prebuilt_file` at this level, so the argument should be like `prebuilt_file` instead of `prebuilt_package`. Versus is ok, but make them consistent.

Reply:
Call it with named parameters: `node.tools.get(Ltp, prebuilt_file=ltp_prebuilt_file, git_tag=ltp_tests_git_tag)`